scripts/deploymentScript.js

```diff
@@ -11,27 +11,28 @@ var webapp = {
 var statusBuild = -1;
 var statusStart = -1;
 
+// Don't need to build web app if NODE_ENV is testing
+if (process.env.NODE_ENV !== 'testing') {
```

Comment:
Note to self, @amoshydra

This checking doesn't actually work, Travis actually still run without using NODE_ENV `testing`.
Special instruction need to be added to `travis.yaml` or `package.json` for the script to detect travis environment. #260